src/main/kotlin/no/nav/medlemskap/domene/Medlemskap.kt

```diff
@@ -18,6 +18,26 @@ data class Medlemskap(
 
 }
 
+enum class Lovvalg(val lovvalgKodeverdi: String) {
```

Comment:
Endrer det på noen måte det vi sender fra oss til konsumentene?

Reply:
God observasjon, det tenkte jeg ikke på. Sjekket nå og verdien endres som følge av endringen. Konsumentene parser ikke alle verdiene, så jeg regner med at det hadde gått bra med prodsetting. Men det er vel kanskje ikke helt optimalt at jeg endrer på en kodeverdi som kanskje er å regnes som felles. Jeg synes kodeverdien med denne endringen er mer lesbar og forståelig, men det går på bekostning av at vi ikke bruker en kodeverdi som er felles. I dette tilfelle blir det kanskje mest riktig å gjøre at det i responsen gis den gamle kodeverdien, men at jeg beholder den bedre og mer lesbare verdien internt?

Reply:
Hvordan så det ut før, det vi returnerte? Det er godt mulig det går greit å endre, SP bryr seg vel ikke, Dagpenger aner jeg ikke.. Vi kan høre med dem. Forøvrig er dette et problem for oss som vi bør løse, vi trenger et lag "utenpå" som gjør at vi står friere til å endre det vi trenger internt. Det gjelder ikke bare denne PRen, men generelt.

Reply:
Vi returnerer ENDL, GYLD, UAVK etc i prod nå, mens med denne endringen vil det bli ENDELIG, GYLDIG, UAVKLART etc. Ja, jeg kan spørre Dagpenger for sikkerhetsskyld, dersom det er det vi ønsker å ha som endring, eller om det mest riktige er å ha kodeverdier brukes i MEDL-registeret? Ja, enig i at vi trenger å skille på interne og eksterne klasser. Jeg lager en Trello-oppgave på det.

Reply:
Nå tror jeg at jeg skjønte bedre hva du spurte om.. Jeg tror jeg heller mot at det er bedre at vi returnerer kodeverdiene slik de er i MEDL til konsumentene våre.